shared/params/config.go

```diff
@@ -233,7 +233,10 @@ func DemoBeaconConfig() *BeaconChainConfig {
 	demoConfig.DepositsForChainStart = 8
 	demoConfig.SlotsPerEpoch = 8
 	demoConfig.MinDepositAmount = 100
-	demoConfig.MaxDepositAmount = 3.2 * 1e9
+	demoConfig.MaxDepositAmount = 3.2 * 1e6
```

Comment:
why did we change this to 1e6?

Reply:
Nishant also did this in #2736 , there was a bug where if we initialize genesis state using the beacon config none of the validators would be active since the config wasn't consistent in significant figures. Do I change these back to use `3.2 * 1e9`?

Reply:
no change this back to 1e9, I only did it so that I could test the contract out with a low amount of goeth. the correct amount should be `3.2*1e9`